soql-utils/src/main/scala/com/socrata/soql/util/EncodedError.scala

```diff
@@ -13,10 +15,11 @@ import com.socrata.soql.environment.ScopedResourceName
 // We want our user-actionable error codes to have a common format.  This format will be:
 //
 // {
-//   "code": "some-identifier-for-machines",
+//   "type": "some-identifier-for-machines",
```

Comment:
"message" -> "english" I'm fine with.
"code" -> "type" is ... fine, but also why?

Reply:
Because that's what it was before, and there's code (in dsmapi in particular) that expects to find these names.  Rather than making it willing to accept either, it's nicer to make there be only one format.